.travis.yml

```diff
@@ -13,6 +13,8 @@ branches:
 before_install:
 - echo no | android create avd --force -n test -t $ANDROID_TARGET --abi $ANDROID_ABI
   -c 20M
+- echo yes | android update sdk --filter extra-android-support --no-ui --force > /dev/null
```

Comment:
Moreover, checking the project, seems there is no real dependency on the support library in this repo; I'll double check in command line; in that case, we should fully drop that dependency.